.github/workflows/issue-assistant.yml

```
@@ -643,145 +569,201 @@ jobs:
+        if: ${{ github.event_name == 'issues' }}
+        id: clean-label
+        run: |
+          RAW_OUTPUT='${{ steps.ai-categorize.outputs.response }}'
+          CLEANED=$(echo "$RAW_OUTPUT" | \
+            tr '[:upper:]' '[:lower:]' | \
+            sed -e 's/["'\''$$$$]//g' -e 's/[, ]\+/, /g' -e 's/^[ ,]*//' -e 's/[ ,]*$//')
+          echo "raw_labels=$CLEANED" >> $GITHUB_OUTPUT
+
+      - name: Apply validated labels
+        if: ${{ github.event_name == 'issues' }}
+        uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)